### PR TITLE
[CI:ALL] Bump to v5.2.0-rc2

### PR DESCRIPTION
**Pull request by mheon**
Now with 100% more release notes.

Main has been merged into the branch, but this will be the last time - all further changes in v5.2 will be via cherry-pick.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+
+### Changes
+- Podman now requires the new kernel mount API, introducing a dependency on Linux Kernel v5.2 or higher.
+- Quadlet `.image` units now have a dependency on `network-online.target` ([#21873](https://github.com/containers/podman/issues/21873)).
+- The `--device` option to `podman create` and `podman run` is no longer ignored when `--privileged` is also specified ([#23132](https://github.com/containers/podman/issues/23132)).
+- The `podman start` and `podman stop` commands no longer print the full ID of the pod started/stopped, but instead the user's input used to specify the pod (e.g. `podman pod start b` will print `b` instead of the pod's full ID) ([#22590](https://github.com/containers/podman/issues/22590)).
+- Virtual machines created by `podman machine` on Linux now use `virtiofs` instead of `9p` for mounting host filesystems. Existing mounts will be transparently changed on machine restart or recreation. This should improve performance and reliability of host mounts.
```
Comment (Luap99) [inline RELEASE_NOTES.md:17]: this should point out that that means virtiofsd must be installed on the host and there must be symlink created to the proper location, ref https://github.com/containers/podman/commit/180cc6f86324161c641513f291a1d01434876518

Comment (Luap99) [reply RELEASE_NOTES.md:17]: (but I guess I am fine to leave it for now and add it for the final release or next rc)

**Comment by Luap99** [comment]
/lgtm
/hold

Adding hold so you can merge it at your convenience and push the tag directly after it.

**Comment by mheon** [comment]
/hold cancel
TY

**Pull request merged**